bitnami/tomcat/templates/_pod.tpl

```diff
@@ -178,6 +175,16 @@ volumes:
     emptyDir: {}
   {{- end }}
   {{- end }}
+  {{- if and .Values.metrics.jmx.enabled .Values.metrics.jmx.config (not .Values.metrics.jmx.existingConfigmap) }}
+  - configMap:
+      name: {{ include "tomcat.metrics.jmx.configmapName" . }}
+    name: jmx-config
+  {{- end }}
+  {{- if and .Values.metrics.jmx.enabled .Values.metrics.jmx.existingConfigmap }}
+  - configMap:
+      name: {{ .Values.metrics.jmx.existingConfigmap | quote }}
+    name: jmx-config
+  {{- end }}
```

Comment:
These two can be reduced to a single conditional, the helper `tomcat.metrics.jmx.configmapName` will take care of setting the name of the `existingConfigmap` or the configmap created when providing `metrics.jmx.config`.

```suggestion
  {{- if and .Values.metrics.jmx.enabled (or .Values.metrics.jmx.config .Values.metrics.jmx.existingConfigmap) }}
  - configMap:
      name: {{ include "tomcat.metrics.jmx.configmapName" . }}
    name: jmx-config
  {{- end }}
```

Reply:
Done.